ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -16,14 +16,26 @@ import { updateProgress } from "../../lib/store";
 import { withSnackbar } from "notistack";
 import GenericModal from "../GenericModal";
 import MesheryPerformanceComponent from "./index";
-import { Paper, Typography, Button } from "@material-ui/core";
+import { Paper, Typography, Button,DialogTitle } from "@material-ui/core";
 import fetchPerformanceProfiles from "../graphql/queries/PerformanceProfilesQuery";
 import { withStyles } from "@material-ui/core/styles";
 import { iconMedium, iconXLarge } from "../../css/icons.styles";
 import subscribePerformanceProfiles from "../graphql/subscriptions/PerformanceProfilesSubscription";
 
 const MESHERY_PERFORMANCE_URL = "/api/user/performance/profiles";
-const styles = () => ({
+const styles = (theme) => ({
+  title : {
+    textAlign : 'center',
+    minWidth : 400,
+    padding : '10px',
+    color : '#fff',
+    flexGrow : 1
+  },
+  dialogHeader : {
+    backgroundColor : theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : '#396679',
```

Comment:
Are we not able to reference a theme color for `'#396679',`?

Reply:
Sure thing Lee, 
I have address your change in my latest commit ✅